doc/user/content/sql/system-catalog.md

```diff
@@ -626,7 +626,7 @@ Field       | Type       | Meaning
 Materialize has compatibility shims for the following relations from [PostgreSQL's
 system catalog](https://www.postgresql.org/docs/current/catalogs.html):
 
-  * [`pg_access_methods`](https://www.postgresql.org/docs/current/catalog-pg-access-methods.html)
+  * `pg_access_methods`
```

Comment:
```suggestion
  * [`pg_access_methods`](https://www.postgresql.org/docs/current/catalog-pg-am.html)
```

Reply:
shouldn't I update to:

```suggestion
* [`pg_am`](https://www.postgresql.org/docs/current/catalog-pg-am.html)
```

Reply:
Oh, I edited it right after the initial comment, goddamn notifications. 😅